NuGet.Config

```diff
@@ -8,13 +8,17 @@
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
   <auditSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
   </auditSources>
   <packageSourceMapping>
     <clear />
+    <packageSource key="nuget.org">
+      <package pattern="Microsoft.Net.Sdk.Compilers.Toolset" />
```

Comment:
the `dotnet-public` feed is a curated mirror of nuget.org. You can use that instead, and avoid adding nuget.org as a package source